pyfar/plot/_line.py

```diff
@@ -9,7 +9,7 @@
     MultipleFractionFormatter)
 
 
-def _time(signal, dB=False, log_prefix=20, log_reference=1, unit=None,
+def _time(signal, dB=False, log_prefix=None, log_reference=1, unit=None,
```

Comment:
Time domain always uses 20 as the default :)
```suggestion
def _time(signal, dB=False, log_prefix=20, log_reference=1, unit=None,
```